ansible/eos_install_config

```diff
@@ -42,26 +45,39 @@ requirements:
 options:
     hostname:
         description: IP or FQDN of the device you want to connect to
-        required: true
+        required: True
     username:
         description: Username
-        required: true
+        required: True
     password:
         description: Password
-        required: true
+        required: True
     use_ssl:
```

Comment:
Why not, if it's already there, as long as it is consistent :) we can always delete it in the future, if we get sick of supporting this